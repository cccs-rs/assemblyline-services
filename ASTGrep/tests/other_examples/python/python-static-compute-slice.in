"abc"[1:3]
'1234'[:-2]
'8ftu'[::+-+-(-(+1))]
'iicsanib'[::+-+-(-(+1))]

b'not-a-string'[1:3]
"abc"[::-1]
'nmnnnmnnmnmmnmmnmnm'[::+-+-(-(+1_3_2_3_0_2_7_6_8+(-1_3_2_3_0_2_7_6_7)))]
'do_not_change'[::-_1]
