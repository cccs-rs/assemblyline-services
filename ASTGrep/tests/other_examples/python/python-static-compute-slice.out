'bc'
'12'
'utf8'
'binascii'

b'not-a-string'[1:3]
'cba'
'mnmnmmnmmnmnnmnnnmn'
'do_not_change'[::-_1]
